pkg/apis/pipeline/v1beta1/task_validation_test.go

```diff
@@ -1743,27 +1743,22 @@ func TestSubstitutedContext(t *testing.T) {
 	}
 }
 
-func TestValidateParamArrayIndex(t *testing.T) {
-	stepsInvalidReferences := []string{}
+func TestGetArrayIndexParamRefs(t *testing.T) {
+	stepsReferences := []string{}
 	for i := 10; i <= 26; i++ {
-		stepsInvalidReferences = append(stepsInvalidReferences, fmt.Sprintf("$(params.array-params[%d])", i))
+		stepsReferences = append(stepsReferences, fmt.Sprintf("$(params.array-params[%d])", i))
 	}
-	volumesInvalidReferences := []string{}
+	volumesReferences := []string{}
 	for i := 10; i <= 22; i++ {
-		volumesInvalidReferences = append(volumesInvalidReferences, fmt.Sprintf("$(params.array-params[%d])", i))
+		volumesReferences = append(volumesReferences, fmt.Sprintf("$(params.array-params[%d])", i))
 	}
 
 	tcs := []struct {
-		name          string
-		params        v1beta1.Params
-		taskspec      *v1beta1.TaskSpec
-		expectedError error
+		name     string
```

Comment:
There are two cases name not updated: "steps reference invalid"->"steps reference"; "stepTemplate reference invalid" -> "stepTemplate reference"